VERSION

```diff
@@ -1 +0,0 @@
-2.0.0
```

Comment:
Why is the VERSION file required in this repo?

Reply:
The VERSION was needed in two places originally, but that has been fixed since so just removing it makes sense 👍